schedule/sles4sap/crash/azure.yml

```diff
@@ -0,0 +1,11 @@
+---
+name: sles4sap_cloud_crash
+description: |
+  crash scenario with one machine for azure
+vars:
+    TEST_CONTEXT: 'OpenQA::Test::RunArgs'
```

Comment:
Is this used anywhere? I see modules are passing context information via openQA settings/variables, not via RunArgs.